programming basics/Python Useful Packages- Numpy.ipynb

```diff
@@ -0,0 +1,1905 @@
+{
```

Comment:
<p>Give a quick one liner here about what this actually means, not just a link.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PTC-CMC/cybercamp/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='64'/>